src/main/java/wordcloud/nlp/FrequencyAnalizer.java

```diff
@@ -48,6 +48,10 @@
     public List<WordFrequency> load(InputStream fileInputStream) throws IOException {
         return load(IOUtils.readLines(fileInputStream));
     }
+    
+    public List<WordFrequency> load(InputStream fileInputStream,FileType fileType) throws IOException {
```

Comment:
I like to add a space after the comma in the parameters.

InputStream fileInputStream, FileType fileType